where-to-travel/src/main/webapp/signup.js

```diff
@@ -77,13 +77,14 @@ function signUp() {
   const promise = firebase.auth().createUserWithEmailAndPassword(emailSignUp.value, passwordSignUp.value);
   promise.then(_ => {      
     // Add user information to the real time database in Firebase
-    let ref = firebase.database().ref('users');
+    let ref = irebase.database().ref('users/' + firebase.auth().currentUser.uid );
     let data = {
+      name: name,
```

Comment:
I think this should be "name: nameSignUp.value" instead of "name: name", and then "const name = document.getElementById("nameSignUp")" can be removed from the top.